src/main/java/org/dinosaur/foodbowl/domain/store/application/StoreService.java

```diff
@@ -0,0 +1,72 @@
+package org.dinosaur.foodbowl.domain.store.application;
+
+import java.util.List;
+import java.util.stream.Collectors;
+import lombok.RequiredArgsConstructor;
+import org.dinosaur.foodbowl.domain.store.dto.StoreRequest;
+import org.dinosaur.foodbowl.domain.store.dto.StoreResponse;
+import org.dinosaur.foodbowl.domain.store.entity.Address;
+import org.dinosaur.foodbowl.domain.store.entity.Store;
+import org.dinosaur.foodbowl.domain.store.repository.StoreRepository;
+import org.dinosaur.foodbowl.global.exception.ErrorStatus;
+import org.dinosaur.foodbowl.global.exception.FoodbowlException;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+@Service
+@RequiredArgsConstructor
+@Transactional(readOnly = true)
```

Comment:
```suggestion
@RequiredArgsConstructor
@Transactional(readOnly = true)
@Service
```
여기도 이런식으로 순서를 배치해보는건 어떨까요 :)